apps/blog/content/posts/what-is-memebattle.en.mdx

```diff
@@ -7,8 +7,106 @@ tags: ['MemeBattle']
 author: 'themezv'
 ---
 
-## Started
+## How it all began
 
+In 2017, a group of students from Novosibirsk State Technical University teamed up to participate in an online hackathon.
+The hackathon was about entertainment and fun. We created an application (web and Android) with an infinity battle of memes. Every few seconds users saw a pair of memes and voted for one of them.
+Players who rated the meme with a majority vote received the internal currency – MemeCoin.
+We won that hackathon and received investments for development.
+
+<details className="my-6">
+  <summary>First application version screenshots</summary>
+  <img src="/content-images/what-is-memebattle/memebattle_v1.png" alt="MemeBattle v1. Screenshow of web application"/>
+  <img src="/content-images/what-is-memebattle/memebattle-playmarket.png" alt="MemeBattle v1. Screenshot of application page on google play"/>
+</details>
+
+After that we have participated in hackathons several years. Local hackathons in Siberia (Tomsk, Kemerovo, Novosibirsk) and grand ones in Moscow – it was our hobby.
```

Comment:
After that, we participated in hackathons for several years. Local hackathons in Siberia (Tomsk, Kemerovo, Novosibirsk) and large ones in Moscow. This was our hobby.